artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.ScrumIntegration/Features/Shared/TimeLineEventParser.cs

```diff
@@ -76,6 +86,83 @@ public GetProjectTimeLineResponse ParseProjectTimeLineElement(IEnumerable<TimeLi
         return new GetProjectTimeLineResponse { TimeLineEvents = timelineEvents };
     }
 
+    private (ScrumObjectType, ScrumObjectState) ParseEventTypeEnum(string eventType)
```

Comment:
Zostawię tylko komentarz który na priv poszedł. Pamiętajmy żeby to potem wydzielić i jakoś otestować